packaging/installer/REINSTALL.md

```diff
@@ -0,0 +1,68 @@
+<!--
+title: "Reinstall the Netdata Agent"
+description: "Troubleshooting installation issues or force an update of the Netdata Agent by reinstalling it using the same method you used during installation."
+custom_edit_url: https://github.com/netdata/netdata/edit/master/packaging/installer/REINSTALL.md
+-->
+
+# Reinstall the Netdata Agent
+
+In certain situations, such as needing to enable a feature or troubleshoot an issue, you may need to reinstall the
+Netdata Agent on your node.
+
+Before you try reinstalling Netdata, figure out which [installation method you
+used](/packaging/installer/UPDATE.md#determine-which-installation-method-you-used) if you do not already know. This will
+deterimine the reinstallation method.
+
+## One-line installer script (`kickstart.sh`)
+
+Run the one-line installer script with the `--reinstall` parameter to reinstall the Netdata Agent. This will preserve
+any [user configuration](/docs/configure/nodes.md) in `netdata.conf` or other files.
+
+If you used any [optional
+parameters](/packaging/installer/methods/kickstart.md#optional-parameters-to-alter-your-installation) during initial
+installation, you need to pass them to the script again during reinstallation. If you cannot remember which options you
+used, read the contents of the `.environment` file and look for a `REINSTALL_OPTIONS` line. This line contains a list of
```

Comment:
As this becomes very technical. Is it possible to show an example of .environment?

Reply:
Should be possible to pull the contents from any arbitrary install. It’s not like it contains any sensitive data.

Sample from a static install on one of my systems where nothing was passed to the installer:

```bash
# Created by installer
PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
CFLAGS="-static -O3 -I/openssl-static/include"
LDFLAGS="-static -L/openssl-static/lib"
NETDATA_PREFIX="/opt/netdata"
NETDATA_CONFIGURE_OPTIONS=" --enable-cloud --with-libJudy=externaldeps/libJudy"
NETDATA_ADDED_TO_GROUPS=" adm squid nobody"
INSTALL_UID="0"
NETDATA_GROUP="netdata"
REINSTALL_OPTIONS=""
RELEASE_CHANNEL="stable"
IS_NETDATA_STATIC_BINARY="yes"
NETDATA_LIB_DIR="/opt/netdata/var/lib/netdata"
```

Reply:
Thanks for the example, @Ferroin. I played around with trying to include this somewhere, but every option I tried actually made the doc(s) more complex, as I all of a sudden needed to explain the large code block when all we really need is to explain a single line.

I'm going to go ahead and merge this so that we can get the broader improvements out to users. If we have more continuing issues around the update/reinstall process, then perhaps the `.environment` file deserves a doc of its own, with some additional information about the lines mean, that each of these processes can reference.